sdk/keyvault/azure-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/AzureKeyVaultConfiguration.java

```diff
@@ -6,5 +6,5 @@
 class AzureKeyVaultConfiguration {
     //TODO: Eventually remove these hardcoded strings with https://github.com/Azure/azure-sdk-for-java/issues/3141
     static final String SDK_NAME = "Azure-Keyvault";
-    static final String SDK_VERSION = "4.0.0-preview.6";
+    static final String SDK_VERSION = "4.0.0-preview.6"; // {x-version-update;com.azure:azure-keyvault-certificates;current}
```

Comment:
this should be 4.0.0-preview.5

Reply:
@g2vinay Versions were updated yesterday by @sranthar and the azure-keyvault* artifacts were ticked up from 4.0.0-preview.5 to 4.0.0-preview.6.

Reply:
I'm resolving the conversation. I merged your changed into mine today and all of these versions are what you'd changed them to so we should be good here.